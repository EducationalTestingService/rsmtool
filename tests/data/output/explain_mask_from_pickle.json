[
    [
        4.93806460126142,
        -0.316793975540994,
        4.65591397849462,
        -0.0846667513334603,
        0.8380941502973517
    ],
    [
        5.37989735354046,
        -0.271537693320852,
        4.76267281105991,
        -0.159203084517278,
        0.6196594690655467
    ],
    [
        4.87137322676275,
        -0.3032392174315611,
        4.681992337164751,
        -0.1384091330895669,
        0.1548731492849847
    ],
    [
        4.71849887129509,
        -0.3061862178478969,
        4.42410714285714,
        -0.211288563682129,
        0.5939725872334936
    ],
    [
        4.62497281328427,
        -0.261968415997792,
        5.0,
        -0.121267812518166,
        0.133825193333801
    ]
]