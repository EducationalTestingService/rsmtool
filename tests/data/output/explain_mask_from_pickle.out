5.529429087511419816e+00 0.000000000000000000e+00 4.575396825396829570e+00 -2.041241452319320082e-01 7.885980109448699135e-03
5.056245805348309474e+00 -1.784576525620620024e-01 4.579617834394900555e+00 -1.128665295966199966e-01 2.118105528447014052e-01
4.774912960575191434e+00 -6.495698024616310129e-02 4.953586497890299256e+00 -2.054120075044399973e-01 6.314216803773819997e-01
5.468060141135129726e+00 -2.250175801852050128e-01 3.850210970464130078e+00 -2.598279209846520166e-01 3.695808815443186734e-01
6.269096283706259598e+00 0.000000000000000000e+00 4.276515151515151381e+00 -3.285643703363849144e-01 6.343137397712921288e-01
6.150602768446280066e+00 0.000000000000000000e+00 4.191897654584219701e+00 -1.460204150811419965e-01 9.848580781789207927e-01
